backend/api/views.py

```
@@ -367,6 +367,64 @@
             )
 
 
+class WebcamUploadView(APIView):
+    """Generate SAS upload URL for webcam recordings."""
+    permission_classes = [IsAuthenticated]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)